flowzone.yml

```diff
@@ -44,6 +45,42 @@
       submodules: "recursive"
       token: ${{ secrets.FLOWZONE_TOKEN }}
 
+  # FIXME: https://github.com/balena-io/deploy-to-balena-action/issues/194
+  - &updateBalenaReleaseNotes
+    name: Update balena release notes
+    run: |
+      set -ea
+      # prevent git from existing with 141
+      set +o pipefail
+
+      previous_tag="$(git --no-pager tag --list --sort=-version:refname "v*.*.*" --merged | head -n2 | tail -n1)"
+      release_notes="$(git log ${previous_tag}..${{ github.event.pull_request.head.sha || github.event.head_commit.id }} --pretty=reference | jq -R -s '.')"
```

Comment:
At this point it might be worth templating a new `&getReleaseNotes` step since we are doing it more than once?

The output being a format that could be consumed by both balena deploy and github release jobs?

Reply:
Yeah, that's not a bad idea actually.